frontend/src/index.css

```
@@ -75,6 +75,60 @@ code {
+  overflow: hidden;
+  padding: 2px;
+}
+
+.conic-border-wrapper::before {
+  content: '';
+  position: absolute;
+  top: -150%;
+  left: -150%;
+  width: 400%;
+  height: 400%;
+  background: conic-gradient(transparent 200deg, rgba(255, 255, 255, 0.8), #a78bfa, #6d28d9, transparent);
```

Stylelint (17.2.0)

[error] Expected "0.8" to be "80%" (alpha-value-notation)

(alpha-value-notation)